frontend/src/metabase/databases/utils/schema.ts

```diff
@@ -15,7 +15,7 @@ export const getValidationSchema = (
 
   return Yup.object({
     engine: Yup.string().default(engineKey).required(Errors.required),
-    name: Yup.string().nullable().default(null).required(Errors.required),
+    name: Yup.string().default("null").required(Errors.required),
```

Comment:
Not sure it leads to expected behavior

Reply:
It's not. It's probably the funniest typo I've ever made :)

Reply:
Fixed